### ISSUE TITLE
Use the OS applet to pick a Mii

### ISSUE THREAD

**Issue by s0r00t** [phijor/SpecializeMii#5]
This would allow for a nicer UI, and make progress towards ctrulib. 
However, this applet is currently undocumented.

**Comment by phijor** [comment]
Yes I know. I'm reverse engineering it at the moment! Give it a bit of time, maybe I can get it fully documentented.

**Comment by s0r00t** [comment]
Is there any way to contact you? I am also researching more about this applet.

**Comment by phijor** [comment]
Sure, I can set up a repo on github to track the process. Give me a minute :wink:
